README.md

```diff
@@ -18,6 +18,10 @@ Currently the operator installs the following products:
 - Integreatly solution explorer
 
 ## Prerequisites
+
+NOTE: Due to a change in how networking is configured for openshift in v4.4.6 (mentioned in the [cloud resource operator](https://github.com/integr8ly/cloud-resource-operator#supported-openshift-versions)) there is a limitation on the version of Openshift that RHMI can be installed on for BYOC clusters.
```

Comment:
fyi: This link won't be fully active until this commit is merged. https://github.com/integr8ly/cloud-resource-operator/pull/158/commits/702a2a863ee5fa0ce4bb49d7dd4130319d4ccd56